prisma/schema.prisma

```diff
@@ -2,14 +2,14 @@
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
 
 generator client {
-    provider      = "prisma-client-js"
-    binaryTargets = ["native", "debian-openssl-1.1.x"]
+  provider      = "prisma-client-js"
```

Comment:
I'm guessing this file wasn't formatted?

Reply:
Some of the linting may have changed. Please make a separate pr to fix the style to separate logic and formatting changes